packages/eslint-config/package.json

```diff
@@ -24,7 +24,68 @@
   "devDependencies": {
     "eslint": "^7.29.0",
     "eslint-plugin-import": "^2.23.4",
+    "eslint-plugin-unicorn": "^40.1.0",
     "typescript": "^4.3.2"
   },
-  "private": false
+  "private": false,
+  "eslintConfig": {
```

Comment:
The eslint rules should be defined in the `eslintrc.js` file, along with the rest of the configs. 

Also, if you can merge the main branch and resolve the conflicts (and probably move `eslint-plugin-unicorn` in `dependencies` 😉

Reply:
Thanks for moving the config, but I still see the eslintConfig object here

Reply:
Sorry! Forgot to delete it after copying the rules over